### PR TITLE
[depends] Readd option to build without dvdcss after #9120

### PR DESCRIPTION
**Pull request by Rechi**
Maybe cmake needs also to be adopted.

### CODE REVIEW

File: tools/depends/target/libdvdread/Makefile [hunk 1]
```diff
@@ -16,14 +16,21 @@ else
   ifeq ($(PLATFORM),)
     PLATFORM = native
     TARBALLS_LOCATION = $(ROOT_DIR)
-    PKGCONFIGPATH = PKG_CONFIG_PATH="$(ROOT_DIR)/../libdvdcss/dvdcss-install/lib/pkgconfig"
-    EXTRA_CFLAGS += -I$(ROOT_DIR)/../libdvdcss/dvdcss-install/include
+    ifneq ($(BUILD_DVDCSS),0)
```
Comment (wsnipex) [inline tools/depends/target/libdvdread/Makefile]: this disables DVDCSS support globally for every platform that builds with depends as BUILD_DVDCSS is not defined in depends.

**Comment by wsnipex** [comment]
I removed this stuff, because it is pretty difficult to get right for all platforms and imho not worth the hassle.
Since the only reason to build without dvdcss is licensing/copyright, distros are free to patch it out.

File: tools/depends/target/libdvdread/Makefile (hunk 1 above)
Comment (Rechi) [inline tools/depends/target/libdvdread/Makefile]: I tested again on osx and linux and if `BUILD_DVDCSS`is not set, it will build with with libdvdcss.

**Comment by FernetMenta** [comment]
I agree with @wsnipex 
We don't build without dvdcss, hence no reason for the extra code.

**Comment by MartijnKaijser** [comment]
Guess we can one this then

**Pull request closed without merging**